pom.xml

```diff
@@ -104,6 +104,18 @@
       <version>${assertj-core.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.jparams</groupId>
+      <artifactId>jparams-junit4</artifactId>
+      <version>1.0.4</version>
```

Comment:
Please define a version in the `properties` section a reference it here, so that all of the versions would be controlled from one place